CHANGES.rst

```diff
@@ -6,6 +6,9 @@ New Features
 
 - Profile viewers now support plotting with profiles "as steps". [#1595]
 
+- For JWST products, line flux in the Line Analysis plugin are reported 
+  in base units of W/m2. [#1564]
```

Comment:
does this need updating now that its a more general solution than just JWST?

Reply:
Yes! Thanks for the catch. Fixing....